examples/demo/infrastructure_provisioning_with_storage.rb

```diff
@@ -0,0 +1,94 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# Be able to provision compute (with server settings), networking, and storage.
+#	Create a server profile template with the following options:
+#		Network connections
+#		Boot mode
+#		Boot settings
+#		Shares volume from the provided storage
+#		Create a server profile from a server profile template and assign to hardware
+#		Power on server
+# This example works for api_variant "Synergy".
+# This example works with either resource uri or resource name.
+# This example has been tested againts Oneview API Version 800.
+
+my_client = {
+  url: ENV['ONEVIEWSDK_URL'],
+  user: ENV['ONEVIEWSDK_USER'],
+  password: ENV['ONEVIEWSDK_PASSWORD'],
+  api_version: 800
```

Comment:
modify the API version with the latest against which this use case has been tested